Chocolatey.Cake.Recipe/Content/parameters.cake

```diff
@@ -348,7 +348,6 @@ public static class BuildParameters
         BuildSystem buildSystem,
         DirectoryPath sourceDirectoryPath,
         string title,
-        string certificateSubjectName = null,
```

Comment:
Just curious... is there a reason that that ability to change this via the recipe.cake file has been removed?

Reply:
Apologies, I didn't fully understand that that was what this was doing. I was modelling my change off of how `CertificatePassword` and `CertificateFilePath` were being set

Reply:
I've pushed an updated commit just now that retains that ability to change it via the recipe.cake file.

Reply:
> @Windos said...
Apologies, I didn't fully understand that that was what this was doing. I was modelling my change off of how CertificatePassword and CertificateFilePath were being set

I think part of the issue here is that we don't have consistency in how these things are being set.  It is one of those things that has evolved over time as the needs have arisen.  At some point, we need to go back through everything and make them consistent, bringing along the learnings into those areas.